torch/xla/__init__.py

```diff
@@ -0,0 +1 @@
+from . import amp
```

Comment:
I don't think these `torch/xla/*` files should live in core. instead, any `torch.xla` autocast related API's can live in the pytorch/xla repo, and can be installed when you import `torch_xla`.

Reply:
xla/* files have been removed.